### PR TITLE
update packages

### PR DESCRIPTION
**Pull request by damip**
* [ ] document all added functions
* [ ] try in sandbox /simulation/labnet
  * [ ] if part of node-launch, checked using the `resync_check` flag
* [ ] unit tests on the added/changed features
  * [ ] make tests compile
  * [ ] make tests pass 
* [ ] add logs allowing easy debugging in case the changes caused problems
* [ ] if the API has changed, update the API specification

### CODE REVIEW

**Comment by damip** [comment]
@aoudiamoncef can you take a look at the remaining compilation errors ?

**Comment by aoudiamoncef** [comment]
Yep, I need to test this and fix the eventual breaking changes.

**Comment by aoudiamoncef** [comment]
hi @damip 

We have to upgrade `massa-proto-rs` 

I copied from Léo PR the breaking change from `jsonrpsee` 

TO be able to merge this PR, we should switch to `rev` for `massa-proto-rs` and pointe to https://github.com/massalabs/massa-proto-rs/pull/28

**Comment by aoudiamoncef** [comment]
@damip we are blocked by `massa-sc-runtime`. I got issues with abi stuff

File: Cargo.toml
Comment (litchipi) [inline Cargo.toml:123]: Is this backport made on purpose ?

File: Cargo.toml
Comment (litchipi) [inline Cargo.toml:113]: Ancient version used here as well

File: Cargo.toml
Comment (damip) [reply Cargo.toml:123]: it's not a backport: https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html

File: Cargo.toml
Comment (damip) [reply Cargo.toml:113]: see https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html

File: Cargo.toml
Comment (Leo-Besancon) [inline Cargo.toml:102]: Not sure what rev. we want, but these commits hashes are not the latest on main.

**Review by Leo-Besancon — APPROVED** [review]
Approved with one check to make.

**Review by litchipi — DISMISSED** [review]
Approved as long as the backward versionning is made on purpose on Cargo.toml

**Review by AurelienFT — COMMENTED** [review]
We should add the update of wasmer in massa-sc-runtime that fix the bug on M1/M2 also no ? https://github.com/massalabs/massa/issues/4359#issuecomment-1718796281

**Comment by damip** [comment]
> We should add the update of wasmer in massa-sc-runtime that fix the bug on M1/M2 also no ? [#4359 (comment)](https://github.com/massalabs/massa/issues/4359#issuecomment-1718796281)

yes we can use the opportunity

File: Cargo.toml
Comment (AurelienFT) [reply Cargo.toml]: They are now

Comment (Leo-Besancon) [reply Cargo.toml]: I don't think so for proto-rs.
But cc0b0b is the commit in massa-sc-runtime, so either we update it in both, or we let it like that.
https://github.com/massalabs/massa-proto-rs/commits/main

**Comment by AurelienFT** [comment]
@Leo-Besancon everything is update

**Review by Leo-Besancon — APPROVED** [review]

**Comment by AurelienFT** [comment]
bors merge

**Pull request merged**
